src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -239,6 +239,9 @@ def safe_prepare(self, args=None):
         """
         if args is not None:
             self.args = args
+        if api.is_ceph_device(self.get_lv(self.args.data)):
+            logger.info("device {} is already used".format(self.args.data))
+            raise RuntimeError("skipping {}, it is already prepared").format(self.args.data))
```

Comment:
A syntax error sneaked in.
```suggestion
            raise RuntimeError("skipping {}, it is already prepared".format(self.args.data))
```

Reply:
:flushed: